init-letsencrypt.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+domains=(ticket.simplificator.com)
+rsa_key_size=4096
+data_path="./data/certbot"
+email="operations@simplificator.com" # Adding a valid address is strongly recommended
+nginx_service_name=zammad-nginx
+compose_files=" -f docker-compose.yml -f docker-compose.prod.yml "
+staging=0 # Set to 1 if you're testing your setup to avoid hitting request limits
+
+if [ -d "$data_path" ]; then
+  read -p "Existing data found for $domains. Continue and replace existing certificate? (y/N) " decision
+  if [ "$decision" != "Y" ] && [ "$decision" != "y" ]; then
+    exit
+  fi
+fi
+
+
+if [ ! -e "$data_path/conf/options-ssl-nginx.conf" ] || [ ! -e "$data_path/conf/ssl-dhparams.pem" ]; then
```

Comment:
Is it safe to do that? In other projects, we hardcode the SSL options in the nginx configuration and generate Diffie-Hellman parameters locally.

Reply:
the tutorial i used proposes to use the recommended parameters by certbot. which i find a good idea.

the diffie-hellman parameters are not a secret. according to https://security.stackexchange.com/questions/94390/whats-the-purpose-of-dh-parameters

Reply:
Sorry, I did not state my concern correctly. Do we accept the following risks?

- GitHub being down or repository/files becoming somehow unavailable or compromised
- Not having the SSL options under version control

It's an acceptable risk for me, but I wanted to make sure it was considered.

Reply:
ok. lets keep the downloaded files under version control. 
but leave the init-script as is for documentation.